addons/web/static/tests/views/list_view_tests.js

```diff
@@ -1732,6 +1732,16 @@ QUnit.module("Views", (hooks) => {
         assert.strictEqual(target.querySelector(".o_data_cell").getAttribute("data-tooltip"), null);
     });
 
+    QUnit.only("text field has no title (data-tooltip)", async function (assert) {
```

Comment:
QUnit.test

Also, you can squash this commit with the first one.

Besides thats, test is ok :-)

Reply:
yes obviously ^^'
thanks for the review